Bottom: 3ea8742250fc0325e7cdf4048ee5cf3949ed6c2f
Top:    8c58bd02a82bdaffaa1a904955271cf93abd08f2
Author: student <student@ReliableEmbeddedSystems.com>
Date:   2019-09-10 21:47:58 +0200

thud as warrior

Signed-off-by: student <student@ReliableEmbeddedSystems.com>


---

diff --git a/meta-mender-core/conf/layer.conf b/meta-mender-core/conf/layer.conf
index d7352bd..123a319 100644
--- a/meta-mender-core/conf/layer.conf
+++ b/meta-mender-core/conf/layer.conf
@@ -17,4 +17,4 @@ BBFILE_PRIORITY_mender = "6"
 
 INHERIT += "mender-setup"
 
-LAYERSERIES_COMPAT_mender = "thud"
+LAYERSERIES_COMPAT_mender = "thud warrior"
